app/src/main/java/com/hover/stax/settings/SettingsFragment.kt

```diff
@@ -113,6 +114,13 @@ class SettingsFragment : Fragment(), NavigationInterface {
         }
     }
 
+    private fun setupLearnCard() {
```

Comment:
The learn card isn't showing up in settings for me.

Reply:
New push should fix